src/scss/styles.scss

```diff
@@ -0,0 +1,50 @@
+// Bootstrap styles overrides
+@import "variables";
+
+// Bootstrap styles
+@import "../../node_modules/bootstrap/scss/bootstrap.scss";
+
+// Fonts
+@import url("https://use.typekit.net/gpj3tvv.css");
+
+.row {
+  margin: 0;
```

Comment:
This is not recommended. The row system uses margins to create a reliable responsive layout. Changing these without overriding the relevant variable might give unwanted results.